util/GO9PTERM

```diff
@@ -16,8 +16,11 @@ if [ "$(uname)" = "Linux" ] && [ -e /dev/kvm ]; then
 	fi
 fi
 
+# specific usb devices added by '-device usb-host,hostbus=X,hostaddr=X'
+# replacing 'X' with the device numbers provided by a tool like 'lsusb'
 read -r cmd <<EOF
 $kvmdo qemu-system-x86_64 -s -cpu Opteron_G1 -smp 2 -m 2048 $kvmflag \
+-usb \
```

Comment:
could you put some examples of usb usage in here? I don't understand this.
Maybe you could how you add your mouse by finding it with lsusb and then adding it to the command line or something